pkg/features/embedded/boot-assessment/usr/sbin/elemental-boot-assessment

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+declare checkResultsPath="/run/elemental/boot-assessment"
+declare checkersPath="/usr/libexec/elemental-checker"
```

Comment:
Any script/executable included in this folder will be executed with  `check` argument. If it returns no error marks the script as passed (so it won't be executed again in any eventual retry), if it returns an error raises the error flag and continues with the rest of the checks. In case any of the executed scripts fails this script runs an `exit 1` which causes the service to fail and restart again after 30seconds.